component/src/lib.rs

```diff
@@ -215,4 +224,17 @@ targets = ["linux_amd64", "linux_arm64", "darwin_amd64", "darwin_arm64"]
     fn test_collect_plugin_executables() {
         assert!(Components::collect_plugin_executables().is_ok());
     }
+
+    #[test]
+    fn test_is_default_forc_plugin() {
+        let executables = Components::collect_plugin_executables().unwrap();
+
+        for executable in executables.iter() {
+            if ["forc-lsp", "forc-doc", "forc-client", "forc-fmt"].contains(&executable.as_str()) {
```

Comment:
Do we not have consts for these?

Reply:
Hmm, on second thought I might just remove this test because it's basically just testing a bool, which isn't very valuable, but the consts were removed when we introduced the `Component` abstraction which basically got rid of all the const strs that we were using.